src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int iterations = 3; //How many times you want to play lottery
```

Comment:
better remove all comments from the code, your code should be understandable without then